ext/smoke/repos/steps/run-validation-tests.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash
+
+set -e
+
+function on_host() {
```

Comment:
These validation tests could be re-used too. They're pretty much the same. I think I just got lazy and wanted to copy-paste to put things in the same directory. Sorry :)